src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -320,6 +333,18 @@ public override IEnumerable<Cci.IMethodDefinition> GetMethods(EmitContext contex
             return method;
         }
 
+        internal Cci.INamespaceTypeDefinition? GetType(string name)
```

Comment:
>GetType [](http://example.com/codeflow?start=47&length=7)

nit: naming this `GetSynthesizedInlineArrayType` would make more sense. `GetType` is confusing/misleading. Same comment for `TryAddSynthesizedType`. Alternatively, we can rename `_synthesizedInlineArrayTypes` to something less specific like `_additionalTopLevelTypes`.